raid.go

```diff
@@ -72,12 +76,11 @@ func parseMdstat(data string) (RaidArrays, error) {
 			for i := 0; i < len(matches[1]); i++ {
 				if matches[1][i:i+1] == "_" {
 					raid.Failed = append(raid.Failed, i)
+				} else if matches[1][i:i+1] == "U" {
```

Comment:
Please add some comment what this check is doing

Reply:
done